### DISCUSSION TITLE
Allow scheduling of Mod Installs/Uninstalls (on next restart, etc)

### DISCUSSION THREAD

**Discussion by TheHoboArk** [gameserverapp/Platform#389] [Suggestions]
Currently, you can only install or uninstall a mod while the server is shutdown. In order not to inconvenience the end-player, it would helpful to schedule an install or an uninstall action on a mod, so that the next time the server naturally restarts, it adds/removes the mod automatically.

**Comment by ssiwm** [comment]
Awesome idea. ASM has this feature.  I miss that a lot in GSA.

**Comment by birdman3131** [comment]
This might be useful in other games but in Ark given how any mod update breaks clients that connect I would say set the update to give a full hour of warning. Even boss fights top out at I think 30 minutes.

Any more time than that and offline people can no longer join because steam auto updated their mods. (And yet it won't auto update when you want it to but thats a different rant.)

**Comment by mvaessen** [comment]
Thanks for the suggestions folks!

This is now available: https://docs.gameserverapp.com/dashboard/game_servers/mods_plugins/#update--delete
